task_3/theater-no-orm/pkg/data/theater.go

```diff
@@ -0,0 +1,427 @@
+package data
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type Account struct {
+	Id          int
+	FirstName   string
+	LastName    string
+	PhoneNumber string
+	Email       string
+}
+
+type Ticket struct {
+	Id                  int
+	PerformanceName     string
+	GenreName           string
+	PerformanceDuration string
+	DateTime            string
+	HallName            string
+	HallCapacity        int
+	LocationAddress     string
+	LocationPhoneNumber string
+	SectorName          string
+	Place               int
+	Price               int
+	DateOfIssue         string
+	Paid                bool
+	Reservation         bool
+	Destroyed           bool
+}
+
+type TicketInsert struct {
+	Id          int
+	AccountId   int
+	ScheduleId  int
+	PlaceId     int
+	DateOfIssue string
+	Paid        bool
+	Reservation bool
+	Destroyed   bool
+}
+
+type Poster struct {
+	Id                  int
+	PerformanceName     string
+	GenreName           string
+	PerformanceDuration string
+	DateTime            string
+	HallName            string
+	HallCapacity        int
+	LocationAddress     string
+	LocationPhoneNumber string
+	Comment             string
+}
+
+type User struct {
+	Id                  int
+	FirstName           string
+	LastName            string
+	Role                string
+	LocationAddress     string
+	LocationPhoneNumber string
+	PhoneNumber         string
+}
+
+type UserInsert struct {
+	Id          int
+	AccountId   int
+	FirstName   string
+	LastName    string
+	RoleId      int
+	LocationId  int
+	PhoneNumber string
+}
+
+type Hall struct {
+	Id         int
+	AccountId  int
+	Name       string
+	Capacity   int
+	LocationId int
+}
+
+type Location struct {
+	Id          int
+	AccountId   int
+	Address     string
+	PhoneNumber string
+}
+
+type Performance struct {
+	Id        int
+	AccountId int
+	Name      string
+	GenreId   int
+	Duration  string
+}
+
+type Place struct {
+	id       int
+	SectorId int
+	Name     string
+}
+
+type PosterInsert struct {
+	Id         int
+	AccountId  int
+	ScheduleId int
+	Comment    string
+}
+
+type Price struct {
+	Id            int
+	AccountId     int
+	SectorId      int
+	PerformanceId int
+	Price         int
+}
+
+type Role struct {
+	Id   int
+	Name string
+}
+
+type Schedule struct {
+	Id            int
+	AccountId     int
+	PerformanceId int
+	Date          string
+	HallId        int
+}
+
+type Sector struct {
+	Id   int
+	Name string
+}
+
+type Genre struct {
+	Id   int
+	Name string
+}
+
+type TheaterData struct {
```

Comment:
also separate structs for methods. So, instead of `TheaterData` should be `AccountData` ,`TicketData`